uniffi_bindgen/src/bindings/kotlin/mod.rs

```diff
@@ -59,9 +59,8 @@ pub fn compile_bindings(config: &Config, ci: &ComponentInterface, out_dir: &Path
     let status = Command::new("kotlinc")
         // Our generated bindings should not produce any warnings; fail tests if they do.
         .arg("-Werror")
-        // Reflect $CLASSPATH from the environment, to help find `jna.jar`.
         .arg("-classpath")
-        .arg(env::var("CLASSPATH").unwrap_or_else(|_| "".to_string()))
+        .arg(classpath_for_testing(out_dir)?)
```

Comment:
huh - I didn't realize `compile_bindings` was only used for tests - do you mind adding a comment in the block at the top of the function noting that?